### PR TITLE
Chat feature

### PR DESCRIPTION
**Pull request by djkaty**
Lets users chat with each other in Hearthstone :)

### CODE REVIEW

File: pegasus/whisper.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	notifier.Notify(&notification_service.Notification{
+		Type: proto.String(bnet.NotifyWhisper),
+		TargetId: &targetId,
+		TargetAccountId: session.AccountEntity,
+		SenderId: s.host.GameAccountEntity,
+		SenderAccountId: s.host.AccountEntity,
+		SenderBattleTag: &s.host.Account.BattleTag,
+		Attribute: bnet.NewNotification("", map[string]interface{} { "whisper": message }).Attributes,
+	})
+}
```
Comment (Mischanix) [inline pegasus/whisper.go]: I want this newline

**Comment by Mischanix** [comment]
LGTM otherwise

File: pegasus/whisper.go (hunk 1 above)
Comment (jleclanche) [inline pegasus/whisper.go]: My man.

Comment (djkaty) [inline pegasus/whisper.go]: I will fix it :D

**Pull request merged**